fuel/transformers/__init__.py

```diff
@@ -75,8 +77,8 @@ class Mapping(Transformer):
         data under source names `add_sources`.
 
     """
-    def __init__(self, data_stream, mapping, add_sources=None):
-        super(Mapping, self).__init__(data_stream)
+    def __init__(self, data_stream, mapping, batch=False, add_sources=None):
```

Comment:
It's probably better to use `...add_sources=None, **kwargs)` and then do `super(Mapping, self).__init__(data_stream, **kwargs)`.